remotes/fixup.go

```diff
@@ -220,59 +220,69 @@ func fixupBaseImage(ctx context.Context, name string, baseImage *bundle.BaseImag
 		info, pushed, ok, err := f(ctx, targetRepoOnly, baseImage, cfg)
 		if err != nil {
 			log.G(ctx).Debug(err)
+			return imageFixupInfo{}, false, fmt.Errorf("failed to fixup the image %s for service %q: %v", baseImage.Image, name, err)
 		}
 		if ok {
 			return info, pushed, nil
 		}
 	}
 
-	return imageFixupInfo{}, false, fmt.Errorf("failed to resolve or push image for service %q", name)
+	return imageFixupInfo{}, false, fmt.Errorf("failed to resolve or push image %s for service %q", baseImage.Image, name)
```

Comment:
This is the one error message that I'm not clear on how we actually get into this situation and what else we could include in the message to help someone solve the problem.